### PR TITLE
Update markdown links/config

### PR DESCRIPTION
**Pull request by mmulji-ic**
Http config header check to (hopefully) remove the 403 error for some url's --> didn't work, reverted
Disabled check on 2 links that were causing issues

### CODE REVIEW

File: docs/hub-tutorials/join-mainnet.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ There are many explorers for the Cosmos Hub. For reference while setting up a no
 - [Mintscan](https://www.mintscan.io/cosmos)
 - [Big Dipper](https://cosmos.bigdipper.live/)
 - [Hubble](https://hubble.figment.io/cosmos/chains/cosmoshub-4)
+<!-- markdown-link-check-disable-next-line -->
```
Comment (yaruwangway) [inline docs/hub-tutorials/join-mainnet.md:54]: why not put at the beginning of the file ?

Comment (mmulji-ic) [reply docs/hub-tutorials/join-mainnet.md:54]: I'm disabling just one link, not everything.

Comment (yaruwangway) [reply docs/hub-tutorials/join-mainnet.md:54]: why disable it ?

Comment (mmulji-ic) [reply docs/hub-tutorials/join-mainnet.md:54]: Because it gives a 403 error, site is still available though

**Review by yaruwangway — APPROVED** [review]

**Pull request merged**